content/en/docs/concepts/workloads/controllers/deployment.md

```diff
@@ -1222,5 +1222,7 @@ it is created.
 * `Deployment` is a top-level resource in the Kubernetes REST API.
   Read the {{< api-reference page="workload-resources/deployment-v1" >}}
   object definition to understand the API for deployments.
-* Read about [PodDisruptionBudget](/docs/concepts/workloads/pods/disruptions/) and how
+* [PodDisruptionBudget](/docs/concepts/workloads/pods/disruptions/) and how
   you can use it to manage application availability during disruptions.
+* Using kubectl to [Create a Deployment](/docs/tutorials/kubernetes-basics/deploy-app/deploy-intro/).
+* Interactive Tutorial - [Deploying an App](/docs/tutorials/kubernetes-basics/deploy-app/deploy-interactive/).
```

Comment:
Per [comment](https://github.com/kubernetes/website/issues/38159#issuecomment-1495505827), this is no longer required. Please remove this.

Reply:
```suggestion
```